prov/efa/configure.m4

```diff
@@ -239,6 +238,8 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 
 	AM_CONDITIONAL([ENABLE_EFA_UNIT_TEST], [ test x"$enable_efa_unit_test" != x"" && test x"$enable_efa_unit_test" != x"no" ])
 	AM_CONDITIONAL([HAVE_NEURON], [ test "$have_neuron" = "1" ])
+
+	CPPFLAGS=$save_CPPFLAGS
```

Comment:
GitHub won't let me make a comment on line 189, but I think this undoing of CPPFLAGS should be near the storing of efa_CPPFLAGS (on line 189 right now).  Note that that block is actually probably in the wrong place, since line 219 modifies efa_LIBS.  So you should probably just pull lines 189-196 down here with the CPPFLAGS rollback.  I know it's not directly related to this patch, but might as well fix all the FLAGS handling in one commit?

Reply:
I noticed later that I think the early call of `AC_SUBST(cmocka_rpath)` is probably why fi_info is broken when we build with efa unit tests, but I hadn't noticed the efa_LIBS change.  Thank you.